components/about/AboutOfficerList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="aboutus-table-container">
+    <h2 class="aboutus-table-title">役員一覧</h2>
```

Comment:
Codacy has a fix for the issue: [Expected 1 line break after opening tag (`<h2>`), but no line breaks found.](https://app.codacy.com/gh/AIESEC-in-Japan/aiesecjp-web/pullRequest?prid=7667176)
```suggestion
    <h2 class="aboutus-table-title">
```